lib/DB/ProjectIRDB.cpp

```diff
@@ -53,6 +57,7 @@ ProjectIRDB::ProjectIRDB(IRDBOptions Options) : Options(Options) {
   PB.registerModuleAnalyses(MAM);
   // add the transformation pass ValueAnnotationPass
   MPM.addPass(ValueAnnotationPass());
+  MPM.addPass(FunctionAnnotationPass());
```

Comment:
We probably don't want to instrument every module we analyze. Also, we decided to run our analyses on un-instrumented code, so I would move this instrumentation to somewhere else.